software_steering_council.md

```diff
@@ -0,0 +1,52 @@
+# Software Steering Council
+
+## Purpose
+
+The Jupyter Software Steering Council (SSC) has jurisdiction over software-related decisions across Project Jupyter, with a primary focus on coordination across projects and decisions that have impact across many Jupyter Subprojects. It is also a mechanism for representatives of each project to share information and expertise. Technical decisions and processes where the SSC isn't explicitly involved are automatically delegated to the individual projects to manage their day-to-day activities, create new repositories in their orgs, etc., with independence and autonomy.
```

Comment:
Are we basically going with the current JEP definition of “when a decision requires a JEP vs when it does not”,  and saying that the SSC is only needed for decisions that require a JEP? If so can we make that explicit? It is a bit unclear to me when the SSC must be consulted and when it need not be consulted

Reply:
It's not entirely clear to us what you have in mind here, Chris - the language above indicates that the SSC can participate in all questions that have impact across the project, even if a JEP isn't explicitly written... Not sure more is needed, if you still find lack of clarity could you add more specifics in #103 where we moved the file? Thx! :)

Reply:
The current JEP repository has a [section on when a decision should require a JEP](https://jupyter.org/enhancement-proposals/jupyter-enhancement-proposal-guidelines/jupyter-enhancement-proposal-guidelines.html#what-qualifies-as-a-jep). I am just wondering if we're continuing to use these guidelines.